tst/EnergyPlus/unit/ChillerAbsorption.unit.cc

```diff
@@ -1830,3 +1834,265 @@ TEST_F(EnergyPlusFixture, ChillerAbsorption_Calc)
+                                                        Constant::CWInitConvTemp,
+                                                        state->dataPlnt->PlantLoop(cndLoopNum).FluidIndex,
+                                                        "ChillerAbsorption_Autosize_TEST");
+
+    Real64 Cp_cond = FluidProperties::GetSpecificHeatGlycol(*state,
+                                                            state->dataPlnt->PlantLoop(cndLoopNum).FluidName,
+                                                            thisChiller.TempDesCondIn,
+                                                            state->dataPlnt->PlantLoop(cndLoopNum).FluidIndex,
+                                                            "ChillerAbsorption_Autosize_TEST");
+
+    Real64 const expectedCondVolFlowRate =
+        expectedNomCap * (1.0 + SteamInputRatNom + nomCapToPumpRatio) / (rho_cond * Cp_cond * state->dataSize->PlantSizData(cndLoopNum).DeltaT);
+
+    Real64 const SteamDeltaT = state->dataSize->PlantSizData(genLoopNum).DeltaT;
+    Real64 const Cp_gen = FluidProperties::GetSpecificHeatGlycol(*state,
+                                                                 state->dataPlnt->PlantLoop(genLoopNum).FluidName,
+                                                                 state->dataSize->PlantSizData(genLoopNum).ExitTemp,
+                                                                 state->dataPlnt->PlantLoop(genLoopNum).FluidIndex,
+                                                                 "ChillerAbsorption_Autosize_TEST");
+
+    Real64 const rho_gen = FluidProperties::GetDensityGlycol(*state,
+                                                             state->dataPlnt->PlantLoop(genLoopNum).FluidName,
+                                                             (state->dataSize->PlantSizData(genLoopNum).ExitTemp - SteamDeltaT),
+                                                             state->dataPlnt->PlantLoop(genLoopNum).FluidIndex,
+                                                             "ChillerAbsorption_Autosize_TEST");
+    Real64 const expectedGeneratorVolFlowRate = (expectedNomCap * SteamInputRatNom) / (Cp_gen * rho_gen * SteamDeltaT);
+
+    bool RunFlag(true);
+    Real64 MyLoad(-20000.0);
+
+    Psychrometrics::InitializePsychRoutines(*state);
+    thisChiller.initialize(*state, RunFlag, MyLoad);
+    thisChiller.sizeChiller(*state);
+
+    EXPECT_NE(thisChiller.NomCap, DataSizing::AutoSize);
+    EXPECT_NE(thisChiller.NomPumpPower, DataSizing::AutoSize);
+    EXPECT_NE(thisChiller.EvapVolFlowRate, DataSizing::AutoSize);
+    EXPECT_NE(thisChiller.CondVolFlowRate, DataSizing::AutoSize);
+    EXPECT_NE(thisChiller.GeneratorVolFlowRate, DataSizing::AutoSize);
+
+    EXPECT_GT(thisChiller.NomCap, 0.0);
+    EXPECT_GT(thisChiller.NomPumpPower, 0.0);
+    EXPECT_GT(thisChiller.EvapVolFlowRate, 0.0);
+    EXPECT_GT(thisChiller.CondVolFlowRate, 0.0);
+    EXPECT_GT(thisChiller.GeneratorVolFlowRate, 0.0);
```

Comment:
Size. Ensure it's not autosized and not zero. This is enough to demonstrate #10534 occurs.

Cf commit message in f7967682aa970063d9a0bf40c1c6fc1a87b1ca20

```
Expected equality of these values:
  expectedNomCap
    Which is: 314812.63583550003
  thisChiller.NomCap
    Which is: 0
/home/julien/Software/Others/EnergyPlus/tst/EnergyPlus/unit/ChillerAbsorption.unit.cc:2094: Failure
Expected equality of these values:
  expectedNomCap * nomCapToPumpRatio
    Which is: 1416.65686125975
  thisChiller.NomPumpPower
    Which is: 0
/home/julien/Software/Others/EnergyPlus/tst/EnergyPlus/unit/ChillerAbsorption.unit.cc:2095: Failure
Expected equality of these values:
  expectedEvapVolFlowRate
    Which is: 0.014999999999999999
  thisChiller.EvapVolFlowRate
    Which is: 0
/home/julien/Software/Others/EnergyPlus/tst/EnergyPlus/unit/ChillerAbsorption.unit.cc:2096: Failure
Expected equality of these values:
  expectedCondVolFlowRate
    Which is: 0.030196473750000008
  thisChiller.CondVolFlowRate
    Which is: 0
/home/julien/Software/Others/EnergyPlus/tst/EnergyPlus/unit/ChillerAbsorption.unit.cc:2097: Failure
Expected equality of these values:
  expectedGeneratorVolFlowRate
    Which is: 0.0038118078460457113
  thisChiller.GeneratorVolFlowRate
    Which is: 0
```